build/Dnx.Native.Settings

```diff
@@ -24,7 +24,7 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
       <PreprocessorDefinitions>_CONSOLE;%(PreprocessorDefinitions);$(Defines)</PreprocessorDefinitions>
-      <TreatWarningAsError Condition="'$(TreatWarningsAsErrors)' != ''">true</TreatWarningAsError>
+      <TreatWarningAsError Condition="'$(TreatWarningsAsErrors)' == ''">true</TreatWarningAsError>
```

Comment:
Kind of confused... isn't this now backwards?

1. Why would we enable TWAE only if the setting is *empty*?
2. Why not just force it to always be on? (That is, don't have a setting for it.)